Nodejs/Product/Nodejs/Project/DependencyNodeProperties.cs

```diff
@@ -143,14 +143,6 @@ public class DependencyNodeProperties : NodeProperties {
 
         internal bool IsGlobalInstall {
             get {
-                var node = DependencyNode as HierarchyNode;
-                while (null != node) {
-                    if (node is GlobalModulesNode) {
-                        return true;
-                    }
-
-                    node = node.Parent;
-                }
                 return false;
```

Comment:
Since this expression is now constant, this creates a bunch of dead code paths that could be removed. I'm included to go the "you ain't gonna need it" route and delete these too, but wanted to check first